src/Controller/UI/AccountController.php

```diff
@@ -361,27 +376,27 @@ public function changePasswordAction()
         }
 
         // Send back the set password screen.
-        return $this->render('PelagosAppBundle:Account:changePassword.html.twig');
+        return $this->render('Account/changePassword.html.twig');
     }
 
     /**
      * Post handler to change password.
      *
      * @param Request $request The Symfony Request object.
+     * @param Ldap    $ldap    The Ldap Utility.
      *
      * @throws \Exception When password do not match.
      *
-     * @Route("/change-password")
-     * @Method("POST")
+     * @Route("/change-password", methods={"POST"}, name="pelagos_app_ui_account_changepasswordpost")
      *
      * @return Response A Symfony Response instance.
      */
-    public function changePasswordPostAction(Request $request)
+    public function changePasswordPostAction(Request $request, Ldap $ldap)
     {
         // If the user is not authenticated.
         if (!$this->isGranted('IS_AUTHENTICATED_FULLY')) {
             // User is not logged in, or doesn't have a token.
-            return $this->render('PelagosAppBundle:template:NotLoggedIn.html.twig');
+            return $this->render('template/NotLoggedIn.html.twig');
```

Comment:
Fix twig template extension

Reply:
@praneethpr what needs to happen here?

Reply:
It had the PelagosAppBundle thing before, I fixed it